modules/configuring-scale-bounds-knative.adoc

```diff
@@ -8,8 +8,7 @@
 The `minScale` and `maxScale` annotations can be used to configure the minimum and maximum number of pods that can serve applications.
 These annotations can be used to prevent cold starts or to help control computing costs.
 
-minScale:: If the `minScale` annotation is not set, pods will scale to zero (or to 1 if enable-scale-to-zero is false per the `ConfigMap`).
-
+minScale:: If the `minScale` annotation is not set, pods will scale to zero, or to 1 if `enable-scale-to-zero` is `false` in the ConfigMap.
```

Comment:
s/ConfigMap/`ConfigMap` (in the case of object reference. Otherwise: config map) 

s/pods will scale/pods scale